src/codecs/avif/mod.rs

```diff
@@ -15,9 +15,31 @@ use crate::error::{EncodingError, ParameterError, ParameterErrorKind, Unsupporte
 
 use bytemuck::{Pod, PodCastError, try_cast_slice, try_cast_slice_mut};
 use num_traits::Zero;
-use ravif::{Img, ColorSpace, Config, RGBA8, encode_rgba};
+use ravif::{Img, Config, RGBA8, encode_rgba};
 use rgb::AsPixels;
 
+/// An enumeration over supported AVIF color types
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+pub enum ColorSpace {
+    /// Encode as YCbCr
+    YCbCr,
+    /// Encode as RGB
+    RGB,
+
+    #[doc(hidden)]
+    __NonExhaustive(crate::utils::NonExhaustiveMarker),
+}
```

Comment:
I'm a bit confused with this. Neither YCbCr nor RGB are color spaces, they are very rough methods of specifying an encoding. It seems that `ravif` translates these two specific 'Color config semantics' (quoting avif specification) and in particular uses `BT.709` and `sRGB`. I'd greatly prefer using the sames `Bt709` and `Srgb` for these variants.